rules/default.js

```diff
@@ -5,26 +7,23 @@ module.exports = function (options) {
     // Block level
 
     code : function (code, lang, escaped) {
-        var result;
-
-        // VERY STRANGE CODE, WHY DO WE PROCESS 'javascript' AS IS ?
-        if ('javascript' == lang) {
```

Comment:
maybe then so:
`if (lang == 'bemjson') {`
to insert any bemjson content?

I save "content" block in db, and I want to save any bemjson content too